src/applications/dependents/686c-674/config/chapters/674/addStudentsArrayPages.js

```diff
@@ -460,6 +460,13 @@ export const studentProgramInfoPage = {
         'ui:options': {
           width: 'xl',
         },
+        'ui:validations': [
+          (errors, schoolName) => {
+            if (schoolName?.length > 80) {
+              errors.addError('School name must be 80 characters or less');
+            }
+          },
+        ],
```

Comment:
Please add a unit test for this.

Reply:
I would also ask design about adding a hint to the input about the character limit.